app/jobs/generate_job.rb

```diff
@@ -31,15 +31,15 @@ def perform(exam_template, num_copies, start)
     generated_pdf = CombinePDF.new
     (start..start + num_copies - 1).each do |exam_num|
       m_logger.log("Now generating: #{exam_num}")
-      pdf = Prawn::Document.new(margin: 15) do
-        exam_template.num_pages.times do |page_num|
+      pdf = Prawn::Document.new(margin: 15, skip_page_creation: true) do
```

Comment:
This change makes sense but is there a reason why we're not persisting some of the other layout settings while we're at it? I feel like if we don't then we'll run into other issues in the future where the template files are not copied correctly:

```ruby
pdf = Prawn::Document.new(skip_page_creation: true) do
    template_pdf.pages.each_with_index do |page, page_num|
        start_new_page(size: page.page_size[2..4], layout: page.layout, margins: page.margins.dup)
```

Reply:
Nice catch! This is complicated a little by the fact that we're converting between two PDF formats (Prawn and CombinePDF). I couldn't easily find "margin" data from the CombinePDF pages, but we can investigate that later.

Reply:
I see... that makes sense with the two different libraries. Do you think we should make an issue so we can look into this later. I'll approve this PR for now though.